website/docs/r/stream_analytics_output_servicebus_queue.html.markdown

```diff
@@ -90,7 +90,7 @@ A `serialization` block supports the following:
 
 -> **NOTE:** This is required when `type` is set to `Csv` or `Json`.
 
-* `field_delimiter` - (Optional) The delimiter that will be used to separate comma-separated value (CSV) records. Possible values are ` ` (space), `,` (comma), `   ` (tab), `|` (pipe) and `;`.
+* `field_delimiter` - (Optional) The delimiter that will be used to separate comma-separated value (CSV) records. Possible values are `` (space), `,` (comma), `` (tab), `|` (pipe) and `;`.
```

Comment:
```suggestion
* `field_delimiter` - (Optional) The delimiter that will be used to separate comma-separated value (CSV) records. Possible values are ` ` (space), `,` (comma), `    ` (tab), `|` (pipe) and `;`.
```

Reply:
Thank you for the feedback. The listing of possible values help users to copy pasting those given values. However, I don't see the benefits of listing the literal space and literal tab characters as the [original code](https://raw.githubusercontent.com/hashicorp/terraform-provider-azurerm/1ab58972d7891a5fa098fca0a3026068b67abb9d/website/docs/r/stream_analytics_output_servicebus_topic.html.markdown) will render them into unusable empty code block. Also, see below screenshot taken from the [generated documentation](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/stream_analytics_output_servicebus_topic#field_delimiter) below. Both space and tab code block look exactly the same.
<img width="675" alt="image" src="https://user-images.githubusercontent.com/305414/197960988-3abf8d0e-1e44-46f8-935e-d4fa1a56c250.png">


How about the following workaround? 

* `field_delimiter` - (Optional) The delimiter that will be used to separate comma-separated value (CSV) records. Possible values are (space character), `,` (comma character), (tab character), `|` (pipe character) and `;` (semicolon character).